.travis.yml

```diff
@@ -88,10 +88,13 @@ install:
 script:
   - |
     if [ -z "$STACK_YAML" ]; then
-       cabal new-test
+      cabal new-test
     else
       stack build --test --no-terminal
     fi
+  - grep 'relude ^' summoner.cabal | awk '{print "https://github.com/kowainik/relude/archive/v"$4".tar.gz"}' | xargs curl -sSL | tar xvz -C '/tmp'
+  - cp $(grep 'relude ^' summoner.cabal | awk '{print "/tmp/relude-"$4"/.hlint.yaml"}') .hlint-relude.yaml
```

Comment:
@qoelet Could you please explain, what is going on here? I'm not a bash expert, so explanation of commands is highly appreciated! :slightly_smiling_face:

Reply:
@ChShersh I'm not sure if this would be the best solution going forward yet. I'm grepping the cabal file for the line where `relude ^= x.2x.x`, then grabbing the release version to construct the URL to the tarball, and fetch/untar it to `/tmp`. I then copy the HLint rule file over to the project dir. I'm going to see if/search for a more elegant solution if there is one.

Reply:
@qoelet Keep us updated with the results of your search :+1:

Reply:
@ChShersh there's something I'm reading up on called a sparse checkout in Git; it might be a lot clearer if we use that to show we are fetching the HLint rules off a specific release from relude instead of this bash magic.